src/Queue.php

```diff
@@ -228,7 +228,7 @@ protected function addUris(array $tracks, int $position = null)
 
             $position += $numberOfTracks;
 
-            if ($data["NumTracksAdded"] != $numberOfTracks) {
+            if ($data["NumTracksAdded"] < $numberOfTracks) {
```

Comment:
The `NumTracksAdded` in the response is actually the length of the playlist, so this check fails without any changes. I'm not a fan of this change but without the library having any concept of playlists I'm not sure what else can be done.

Reply:
I've removed this check in 8ac7fbcbcd1a7982f44ce005f3b67107073a97b7 as Sonos will never respond with less tracks than requested